src/App/Components/Header.js

```diff
@@ -0,0 +1,50 @@
+import React from 'react';
+import '../../Assets/scss/style.scss';
+
```

Comment:
the same here we don't need to import the style in the header since its already imported